src/platform/linux/x11/util/randr.rs

```diff
@@ -1,4 +1,5 @@
-use std::slice;
+use std::str::FromStr;
+use std::{slice, env};
```

Comment:
This is the nitpickiest of nitpicks, but I'm trying to cultivate a consistent `use` style.
```rust
use std::{env, slice};
use std::str::FromStr;
```

Reply:
I love it :D. I was just following the default `rustfmt` import ordering.

Reply:
Thanks, but you didn't alphabetize the order of `env` and `slice`.

Reply:
Oh I didn't even notice that, sorry.